gatsby-node.js

```diff
@@ -481,6 +481,11 @@ exports.createPages = async function ({ actions, graphql }) {
       },
     });
 
+    function getLastSegment(url) {
+      let segments = url.split('/');
+      return segments[segments.length - 1];
```

Comment:
```suggestion
      const segments = url.split('/');
      return segments[segments.length - 1];
```

this is never reassigned

Reply:
Actioned in #687